tutorials/export/exporting_for_web.rst

```diff
@@ -232,6 +258,17 @@ player to click, tap or press a key/button to enable audio, for instance when di
 .. warning:: Access to microphone requires a
              :ref:`secure context <doc_javascript_secure_contexts>`.
 
+.. warning::
+
+        By default since Godot 4.3, Web exports will favor the use of samples instead of streams
```

Comment:
```suggestion
        Since Godot 4.3, by default Web exports will favor the use of samples instead of streams
```
These "Since Godot 4.3" notes should be the first thing in their paragraph IMO

```suggestion
        Since Godot 4.3, by default Web exports will use samples instead of streams
```
Can also be more concise